fili-core/src/main/java/com/yahoo/bard/webservice/table/BaseSchema.java

```diff
@@ -29,6 +30,13 @@ protected BaseSchema(Granularity granularity, Iterable<Column> columns) {
     }
 
     @Override
+    public Optional<Column> getColumn(String columnName) {
+        return getColumns().stream()
+                .filter(column -> column.getName().equals(columnName))
+                .findFirst();
+    }
```

Comment:
This has nothing to do with Druid, it has to do with valid states of an object, and what it means to only get the 1st one if there are multiple columns somehow. 

The extra work of going through all the columns to see if there _may_ be others every time a single column is requested seems like massive overhead (ie. if we want that constraint, we should enforce it on construction), and throwing an exception is rather drastic behavior for what seems like an otherwise simple method (to the callers).

Since it's only used in tests, and not production code, and there are clearly more questions than answers here, I'd still vote for simply removing it.